tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js

```diff
@@ -0,0 +1,44 @@
+const CheckboxVisibleTruePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-true.page.js');
+const CheckboxVisibleFalsePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-false.page.js');
+const CheckboxVisibleNonePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-none.page.js');
+const MutuallyExclusivePage = require('../../../generated_pages/checkbox_detail_answer_visible/mutually-exclusive.page.js');
+
+
+describe('Given I start a checkbox survey with a write-in option', function() {
+  beforeEach(function() {
+    browser.openQuestionnaire('test_checkbox_detail_answer_visible.json');
+  });
+
+  it('When I view a write-in checkbox and the visible option is set to true, Then the detail answer label should be displayed', function() {
+    expect($(CheckboxVisibleTruePage.otherDetail()).isDisplayed()).to.equal(true);
```

Comment:
These should use `.to.be.true;` or `.to.be.false;`

Reply:
Done! Thanks👍